ParkingLot/StandardParkingBoy.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParkingLotPlace
+{
+    public class StandardParkingBoy //parking boy act as a proxy here
+    {
+        private List<ParkingLot> parkingLots;
+        private int currentParkingLot;
+
+        public StandardParkingBoy(List<ParkingLot> parkingLots)
+        {
+            this.parkingLots = parkingLots;
+            this.currentParkingLot = 0;
+        }
+
+        public string ParkCar(string car)
+        {
+            while (currentParkingLot < parkingLots.Count)
```

Comment:
WangKe:
There is one kind scenario we should consider that what if we fetch a car in a parkingLot before the currentParkingLot, then the currentParkingLot is not the best parkingLot to park a car as we need to park car sequentially.

Reply:
For the idea of iterating the list to find the right parkingLot, My suggestion is that we look up from the first parkingLot in the list every time.